src/config.rs

```diff
@@ -248,7 +248,7 @@ impl Clock for SystemClock {
     }
 }
 
-fn default_clock() -> Arc<dyn Clock + Send + Sync> {
+pub(crate) fn default_clock() -> Arc<dyn Clock + Send + Sync> {
```

Comment:
Rather than making this pub(crate), is it possible to have the execute bench code do DbOptions::default().clock?